retrodep/vendored.go

```diff
@@ -55,7 +55,8 @@ func processVendoredSource(src *GoSource, search *vendoredSearch, pth string) er
 	thisImport := filepath.ToSlash(filepath.Dir(rel))
 	repoPath, err := src.RepoPathForImportPath(thisImport)
 	if err != nil {
-		return err
+		log.Error(err)
+		return nil
```

Comment:
Why suppress the error return?

Reply:
Ah, I think this is related to OSBS-7505. Any behaviour change here should go in a separate PR I think.

Reply:
Created PR: https://github.com/release-engineering/retrodep/pull/122

Reply:
> Why suppress the error return?

The error is returned by walkfn (line 75), which aborts the whole flow.